docs/repositories/dev.eessi.io.md

```diff
@@ -0,0 +1,9 @@
+# `dev.eessi.io` repository
+
+## What is `dev.eessi.io`?
+
+`dev.eessi.io` is the development repository of EESSI. With it, developers can build and deploy non-production ready versions of their software to a CernVM-FS repository. 
+This way, development version can easily be tested on systems where `dev.eessi.io` is available.
+
+On a system with `dev.eessi.io` mounted access is possible with `module use /cvmfs/dev.eessi.io/versions/2023.06/modules/all`. Then, all that is left is
```

Comment:
This will change when we use subdirectories for projects

Reply:
In fact, we probably (eventually) want a small custom init script for `dev.eessi.io` which:

* prints a warning (something like "`installations in `dev.eessi.io` are not for production use, only for testing`");
* sets up the environment for `software.eessi.io`;
* sets up the environment for a specific (set of) projects in `dev.eessi.io`;